modules/custom-status/custom-status.php

```diff
@@ -414,9 +414,9 @@ function post_admin_header() {
 			// Load the custom statuses
 			foreach( $custom_statuses as $status ) {
 				$all_statuses[] = array(
-					'name' => esc_js( $status->name ),
-					'slug' => esc_js( $status->slug ),
-					'description' => esc_js( $status->description ),
+					'name' => esc_html__( $status->name, 'edit-flow' ),
+					'slug' => $status->slug,
```

Comment:
Could you, please, elaborate on why removing the `esc_js` function is safe? I mean, the value used to be escaped by `esc_js` which is now being removed form within a commit claiming that is fixing localisation issues.

This feels like unrelated change, so even if it was safe, it ideally should be a standalone commit.